libs/wingii/src/lib.rs

```diff
@@ -48,20 +53,45 @@ pub mod spec {
 		}
 	}
 
+	fn try_load_from_cache(hash: &str) -> Option<Assembly> {
+		let file_path = format!("{CACHE_FILE_DIR}/{hash}.{CACHE_FILE_EXT}");
```

Comment:
We should also support bundling the cached version of the jsii manifest into the library.

For example, if there's a file called `wing.types` (or something like that) inside the npm module (next to `.jsii`), then use it instead of trying to load `.jsii`. Then we can create this file during build and bundle it into `@winglang/sdk/wing.types`.

(we will also need some tool that can generate this `.types` file so we can use it in our build).

(not sure about the name)

Reply:
In practice the only .jsii files we have are wingsdk's and constructs, both of which load pretty quickly and also serialize to the cache pretty quickly so I don't think it'll make much of a difference and not worth the effort.
Pre-caching cdktf/awscdk makes sense, but they aren't part of our bundle, so I think it's out of this scope, and more related to adding support for binary manifests in .JSII (should I do a PR there?).